src/client.h

```diff
@@ -121,8 +120,11 @@ class CClient : public QObject
 
     void Start();
     void Stop();
+    bool Connect ( QString strServerAddress, QString strServerName );
```

Comment:
It does make sense to introduce an easy to call Connect() and Disconnect() method IMO, so this is good.

Reply:
Having these in `client.cpp` rather than `clientdlg.cpp` is important.  It's important enough to be done in a single PR on its own. I think we should start with that.

Reply:
Agree. That's issue https://github.com/jamulussoftware/jamulus/issues/3367

Reply:
Factored out into https://github.com/jamulussoftware/jamulus/pull/3372